### DISCUSSION TITLE
SDKs?

### DISCUSSION THREAD

**Discussion by dannysheridan** [keygen-sh/community#54] [General]
Have you considered offering SDKs for Keygen?

**Comment by ezekg** [comment]
We do have [a Go SDK](https://github.com/keygen-sh/keygen-go). Others are on the backlog. What language SDK are you looking for?

**Comment by dannysheridan** [comment]
What led me here: my team is looking for a Timed License API Key Management solution.

What kept me here: our business is providing SDKs as a service. We produce Java, Python, and Node.JS SDKs. You define your API once (in YAML) and then use that to codegen SDKs. Curious what SDK languages/frameworks are in your backlog!

**Comment by ezekg** [comment]
@dannysheridan Our SDK backlog includes Rust, Python, C#, Node, Kotlin and Swift. But unfortunately, I'm not sure if codegen'd SDKs would offer the DX we're after for here. If you look at our Go SDK, it's not a 1-1 mapping of our API, it's more tailored to the Go language and provides a "blessed" flow for accomplishing tasks. But open to being proven wrong on the DX.

Regardless, feel free to shoot me an email to zeke@ and I'll take a look at your business. And more than happy to discuss getting you set up with Keygen as well for licensing your product(s) via timed licenses.
